python/packages/autogen-core/docs/src/conf.py

```diff
@@ -103,6 +103,11 @@
             "url": "https://twitter.com/pyautogen",
             "icon": "fa-brands fa-twitter",
         },
+        {
```

Comment:
LGTM.
Can you add a screenshot of what this looks like, before, after?
Mostly to gauge space utilization

Reply:
added, see pr description @victordibia. should we remove the pypi icon? its redundant with the packages link in the left?

Reply:
Yes. I vote to remove the pypi icon. Thanks!

Reply:
![image](https://github.com/user-attachments/assets/e340dbc3-1f72-442b-8805-748026c2b034)

Reply:
before

Reply:
Is this resolved? Are we good to go?